docs/components/featured-content-modules.md

```diff
@@ -172,21 +158,21 @@ usage: >-
   call-to-action link, and a graphical element related to the piece of content
   being featured. They are a “mini story” and never simply a dominant visual.
 
-  * Headings should be as succinct as possible and 35 characters or less
+  \* Headings should be as succinct as possible and 35 characters or less
   (including spaces).
 
-  * Descriptions should be action-oriented and let the user know what they will
+  \* Descriptions should be action-oriented and let the user know what they will
   get out of the featured content and how it's related to the main page content,
   not just what it is. Descriptions should be 160 characters or less (including
   spaces).
 
-  * Links serve as a call-to-action, so they should begin with an action verb
+  \* Links serve as a call-to-action, so they should begin with an action verb
   like “explore,” “read,” “learn,” or “discover.” When using an action verb, be
   specific and tie the action back to the content and what the user will get
   from it. As a best practice, only include up to one link in a FCM. Link text
   should be 35 characters or less (including spaces).
 
-  * Do not include a link with a video FCM. In a FCM that features video, the
+  \* Do not include a link with a video FCM. In a FCM that features video, the
```

Comment:
The slash here and the ones above mess the formatting up:

<img width="872" alt="Screen Shot 2019-09-26 at 10 55 45 AM" src="https://user-images.githubusercontent.com/704760/65699516-3605d900-e04c-11e9-96c2-3b804803bc26.png">